mypy/options.py

```diff
@@ -34,6 +34,7 @@ class Options:
         "show_none_errors",
         "warn_no_return",
         "warn_return_any",
+        "warn_unused_ignores",
```

Comment:
Oh, now the docs also need to be updated (it has separate sections for global and per-module flags).